backend/app/infrastructure/database/models.py

```diff
@@ -41,6 +41,17 @@ def get_budget(self, epoch: int) -> Optional[int]:
         return int(budget) if budget is not None else None
 
 
+class GPStamps(BaseModel):
+    __tablename__ = "gitcoin_passport_stamps"
+    user_id = Column(
+        db.Integer, db.ForeignKey("users.id"), primary_key=True, nullable=False
+    )
+    score = Column(db.Float, nullable=False)
+    expires_at = Column(db.TIMESTAMP, nullable=False)
+    # storing for analysis
+    stamps = Column(db.String, nullable=True)
```

Comment:
can be null? Is it worth keeping in a db without stamps?

Reply:
Good catch. Actual value that could appear here is an empty list.